docs/guides/development/pack-maker-tools.md

```diff
@@ -0,0 +1,108 @@
+
+
+### Details
+
+Clicking on a log entry expands it to reveal additional details.
+
+<img src="../../../images/pmt_details.png" srcset="../../../images/pmt_details_2x.png 2x" class="screenshot" alt="Log entry details">
+
+These details include information you can use directly (**Pack Version**, etc), as well as internal version that may be useful when working with Coda support (**Request ID**, etc). The **Overview** log entry will include a **Duration** field containing the total execution time.
+
+Failed executions will include some additional detail in the **Overview** entry.
+
+<img src="../../../images/pmt_details_error.png" srcset="../../../images/pmt_details_error_2x.png 2x" class="screenshot" alt="Log entry details for a failed execution">
+
+Specifically the **Error** field will include the error message, and the **Stack trace** field will include the line number where the error happened (`code.ts:18:23` means line 18, character 23).
+
+
+### Searching
+
+You can search for logs using the magnifying glass icon at the top of the panel.
+
+<img src="../../../images/pmt_search.png" srcset="../../../images/pmt_search_2x.png 2x" class="screenshot" alt="Searching for log entries">
+
+Search results are no longer grouped by execution, but instead a flat list of entries from across all executions. You can expand an entry and click the **View related logs** link to view all of the log entries from that execution.
+
+
+## Settings
+
+The Pack maker tools also lets you adjust a few developer-specific settings for the Pack. You can access these settings by clicking the gear icon at the top of the panel.
+
+<img src="../../../images/pmt_settings.png" srcset="../../../images/pmt_settings_2x.png 2x" class="screenshot" alt="Pack settings for the doc">
+
+**Installed in this doc**
+:   Which version of the Pack to use in the doc. This allows you to test new versions of your Pack before releasing them, or roll back to previous versions to reproduce a bug. It's recommended to use **Latest Version** while developing your Pack.
+
+**Auto-refresh formulas & tables**
+:   If you enable this setting and have selected **Latest version** above, whenever you build a new version of your Pack all of the formulas and sync tables in the doc will be automatically refreshed. This only applies to the current session (browser tab), so you'll need to turn it back on if you refresh the page or open the doc again later.
+
+
+## Additional options
+
+A few additional options are available under the three dots menu at the top of the panel. These are mostly quick links that let you jump between the various assets of the Pack, but also small utilities.
```

Comment:
maybe worth flagging the primary action in this menu is the choice to force a refresh of all instances of the pack

Reply:
Done